chpl/chpl-resources/src/main/resources/environment.properties

```diff
@@ -130,3 +130,10 @@ schemaCsv2015NameLegacy=2015 Listing CSV Data Dictionary Legacy.csv
 schemaSurveillanceName=Surveillance and Nonconformity CSV Data Dictionary.csv
 schemaBasicSurveillanceName=Surveillance (basic) CSV Data Dictionary.csv
 #############################################################
+
+######  CRITERIA THAT REQUIRE 170.315 (d)(12) and 170.315 (d)(13)  ######
+requiredCriteriaDependencies=[{\
+    "dependentCriteria": [1,2,3,4,5,9,10,12,13,14,15,16,169,17,170,18,171,21,22,172,23,73,24,174,25,26,27,175,28,30,31,38,40,176,41,42,43,44,45,46,47,177,48,49,56,57,58,179,59,60],\
```

Comment:
Do these IDs include _all_ of the new ones? Or, at least, count all of them?

Reply:
Or maybe is this also blocked by OCD-3283?

Reply:
D2, D3, D10 are included in this list.